mix.exs

```diff
@@ -244,19 +244,18 @@ defmodule FunWithFlags.Mixfile do
         "MIT"
       ],
       links: %{
-        "GitHub" => "https://github.com/tompave/fun_with_flags",
-        "Changelog" => "https://github.com/tompave/fun_with_flags/blob/master/CHANGELOG.md"
+        "GitHub" => @source_url,
+        "Changelog" => "#{@source_url}/blob/master/CHANGELOG.md"
       }
     ]
   end
 
-
   defp docs do
     [
-      extras: ["README.md"],
-      main: "FunWithFlags",
-      source_url: "https://github.com/tompave/fun_with_flags/",
-      source_ref: "v#{@version}"
+      main: "readme",
+      source_url: @source_url,
+      source_ref: "v#{@version}",
+      extras: ["README.md", "CHANGELOG.md"]
```

Comment:
I wouldn't change so many things in this file. I'd leave it as it is, and then maybe use the source URL and add the changelog here.